src/components/Layout/Header/NavItems.tsx

```diff
@@ -28,65 +42,124 @@ type NavItem = NavLink | NavSection
 const NAV_ITEMS: NavItem[] = [
     {
         id: 'topnav',
-        name: 'Products',
+        text: 'Products',
         links: [
             {
-                name: 'Cody',
+                text: 'Cody',
                 href: '/cody',
+                icon: '/home/branded-icons/cody-squircle.svg',
+                subText: 'AI coding assistant with deep codebase context',
+                iconPosition: 'top',
             },
             {
-                name: 'Code Search',
+                text: 'Code Search',
                 href: '/code-search',
+                icon: '/home/branded-icons/Code-Search-squircle.svg',
+                subText: 'Advanced codebase search, batch changes, and insights',
+                iconPosition: 'top',
             },
         ],
     },
     {
         id: 'topnav',
-        name: 'Pricing',
-        href: '/pricing',
-    },
-    {
-        id: 'topnav',
-        name: 'Enterprise',
-        href: '/enterprise',
+        text: 'Solutions',
+        links: [
+            {
+                text: 'Enterprise',
+                subText: 'Scale and security for any size team',
+                href: '/enterprise',
+                icon: (
+                    <div className="hidden w-fit rounded-lg border-1 border-black/25 bg-white p-2 lg:block">
+                        <Building2 className="h-4 w-4 !bg-transparent text-black" />
+                    </div>
+                ),
+                iconPosition: 'top',
+            },
+            {
+                decoratedText: 'Use Cases',
```

Comment:
For the "Use Cases" and "Integrations" section labels, please update the text and underline to be gray-400 at 60% opacity in light mode and full opacity in dark mode.

Reply:
The full opacity looks more readable in terms of accessibility. I think it just needs the underline to match and it will look less "off". Or at least update text and underline to 80% so it has a little more contrast.